compiler/hash-types/src/new/vars.rs

```diff
@@ -0,0 +1,33 @@
+//! Definitions related to variables.
```

Comment:
It seems that there is some confusion about where things belong in `scopes.rs` and `vars.rs`